langchain-core/src/prompts/tests/structured.test.ts

```diff
@@ -90,7 +90,7 @@ test("Test format", async () => {
   const revived: StructuredPrompt = await load(JSON.stringify(prompt));
 
   expect(JSON.stringify(prompt)).toEqual(
-    '{"lc":1,"type":"constructor","id":["langchain_core","prompts","structured","StructuredPrompt"],"kwargs":{"schema_":{"name":"yo","description":"a structured output","parameters":{"name":{"type":"string"},"value":{"type":"integer"}}},"input_variables":[],"messages":[{"lc":1,"type":"constructor","id":["langchain_core","prompts","chat","HumanMessagePromptTemplate"],"kwargs":{"prompt":{"lc":1,"type":"constructor","id":["langchain_core","prompts","prompt","PromptTemplate"],"kwargs":{"input_variables":[],"template_format":"f-string","template":"I\'m very structured, how about you?"}}}}]}}'
+    '{"lc":1,"type":"constructor","id":["langchain_core","prompts","structured","StructuredPrompt"],"kwargs":{"schema_":{"name":"yo","description":"a structured output","parameters":{"name":{"type":"string"},"value":{"type":"integer"}}},"input_variables":[],"messages":[{"lc":1,"type":"constructor","id":["langchain_core","prompts","chat","HumanMessagePromptTemplate"],"kwargs":{"prompt":{"lc":1,"type":"constructor","id":["langchain_core","prompts","prompt","PromptTemplate"],"kwargs":{"input_variables":[],"template_format":"f-string","template":"I\'m very structured, how about you?","schema":{"name":"yo","description":"a structured output","parameters":{"name":{"type":"string"},"value":{"type":"integer"}}}}}}}]}}'
```

Comment:
Why did this change?

Reply:
I believe it's because I started passing optional options through all the prompts, so that `templateFormat` could be passed through. And since I'm now always passing these options through where they weren't always being passed through, schema gets caught in this and is why it's logged. Not entirely sure though.

Is this related to serialization/is breaking?

Reply:
Probably fine if it's additive, but worth checking in with @dqbd or @nfcampos

Reply:
I think it's more that serialization was broken before and this change fixed it!